src/libdir.h

```diff
@@ -7,22 +7,12 @@
 #ifndef _SCOPE_LIBDIR_H
 #define _SCOPE_LIBDIR_H 1
 
-// The base directory is where we will create our Library Directory.
-int         libdirSetBase(const char *basedir); // override default
-const char* libdirGetBase();
+// File types
+#define LIBRARY_FILE 0 // libscope.so
+#define LOADER_FILE 1  // ldscopedyn
```

Comment:
```suggestion
// File types
#define LIBRARY_FILE 0 // libscope.so
#define LOADER_FILE 1  // ldscopedyn
```
Can we use `enum` instead of `int` here? Will be much easier to figure out the first parameter of the functions described below